packages/CopyInput/stories/CopyInput.stories.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { withKnobs } from "@storybook/addon-knobs";
+import { getStoryName } from "storybook/storyTree";
+import Showcase from "./examples/Showcase";
+import WithInputProps from "./examples/WithInputProps";
+
+const storyName = getStoryName("CopyInput");
+
+storiesOf(storyName, module)
```

Comment:
Can we go with the Storybook compact story format (CSF) instead of using `storiesOf`?  

I actually don't mind if you skip the Showcase story entirely though, and put a few examples all in one story.  Whatever makes it easier for you to build the component.

Reply:
Sure, is there an example of a component with CSF stories? Doing a "find all" for CSF doesn't return anything really

Reply:
Sure, I just refactored the `<Radio>` stories, you can check it out.  
https://github.com/acl-services/paprika/tree/master/packages/Radio/stories

You'll need a separate `stories.js` file for each "folder" in the Storybook navigation menu though, because the default export defines that folder.  So if you maintain both of your stories you'll need a `CopyInput.stories.js` and a `CopyInput.examples.stories.js`